src/gc/gcee.cpp

```diff
@@ -543,7 +540,7 @@ void GCProfileWalkHeap()
     }
 #endif // defined (GC_PROFILING)
 
-#ifdef FEATURE_EVENT_TRACE
+#if  defined (GC_PROFILING)//UNIXTODO: Enable this for FEATURE_EVENT_TRACE
```

Comment:
We should remove the TODO comment. This is eventing functionality under GC_PROFILING and expected to just lightup when GC_PROFILING is defined.